src/ApiPort/ApiPort.VisualStudio.Common/Analyze/ProjectAnalyzer.cs

```diff
@@ -83,7 +83,7 @@ public async Task AnalyzeProjectAsync(ICollection<Project> projects, Project ent
 
                 foreach (var file in output)
                 {
-                    targetAssemblies.Add(Path.GetFileName(file));
+                    targetAssemblies.Add(file);
```

Comment:
Do we expect "file" to be fully-qualified? I'm just thinking of other ways this could fail. Could if ever be relative to something other than the working directory?

Reply:
Under debugger, I can see the value of file is absolute path. I'm not sure if in anycase it could be different. Anyhow, the current code total is totally broken. This fix definitly makes it better.